ISISScientificComputing/autoreduce
# Now uses the name of the data file to look for plot files

### Summary of work

Previously it relied on a manual dictionary mapping between instrument name
and regex that will be used to find the name.

This has been removed in favour of just looking for image files of the same
name as the name of the input data file.

This change shouldn't break plotting for MARI or WISH - and no other instruments
currently use plotting at all, so they can pick up this naming scheme.

### How to test your work
Change the SFTP in your `credentials.ini` to the development reduce machine. This should allow you to pick up plots even when the web app is running locally.

- Run web app locally with this change
- Go to http://127.0.0.1:8000/runs/MARI/27037/0/ (assuming you use the default django runserver settings)
- Plots should show, or if they don't there shouldn't be a "encountered error while retrieving plots" error
  - If there is an error - check the SFTP credentials
  - If there are no plots it's possible they were cleaned up (as that data is cleaned daily for dev). Manually remove some runs and resubmit them (I used MARI 27037, perhaps try a WISH run that plots as well, I tried 48104 but there weren't any plots in the reduction output) 

Fixes #794 
Fixes #900 (although we may want more information that the bit I've put in)

**Before merging ensure the release notes have been updated**
Not sure we need release notes - we have stated that plotting should work for all instruments and with this PR it should!
